instrumentation/httpurlconnection/library/src/main/java/io/opentelemetry/instrumentation/library/httpurlconnection/HttpUrlInstrumentationConfig.java

```diff
@@ -119,20 +119,32 @@ public static boolean emitExperimentalHttpClientMetrics() {
         return emitExperimentalHttpClientMetrics;
     }
 
+    /**
+     * Configures the timeout in ms that defines the time that should have elapsed since the
+     * connection was last active. It is used by the idle connection harvester thread to find idle
+     * connections that should be reported. To schedule the idle connection harvester follow
+     * instructions defined in the <a
+     * href="https://github.com/open-telemetry/opentelemetry-android/blob/96ea4aa9fe709838a91811deeb85b0f1baceb8cd/instrumentation/httpurlconnection/README.md#scheduling-harvester-thread">
+     * README</a>.
+     *
+     * <p>Default value: "10000"
+     */
+    public static void setConnectionInactivityTimeoutMs(long timeoutMs) {
+        HttpUrlInstrumentationConfig.connectionInactivityTimeoutMs = timeoutMs;
```

Comment:
should we check if the value here is at least positive and non zero?

Reply:
It does make sense for it to be positive for production workflows. I can throw an IllegalArgumentException if it's negative or zero. Does that make sense? 

We will need another test method than similar to this where we allow for negative values:
 public static void setConnectionInactivityTimeoutMsForTesting(long timeoutMs) {
        HttpUrlInstrumentationConfig.connectionInactivityTimeoutMs = timeoutMs;
}

If the customer wants to test the harvester with this they can as well so no issue in exposing this to the customer?

Reply:
Conventionally, zero has been used for a long time to mean "no timeout, wait forever". Not sure how much we need or care about that here. Just throwing an exception for negative seems reasonable to me.

Reply:
Added an exception for negative values! Also added a test only function to allow negative values for test, so wait times are not needed in test. 

Added some good documentation to prevent misuse. We can also add @VisibleForTesting annotation on the test function. Right now held off on that as I think we're not using the annotation beyond documentation purposes and we need to introduce additional dependency in this module for just one function. Let me know if it's better to have that annotation instead.